testsuite/features/secondary/min_activationkey.feature

```diff
@@ -121,3 +122,6 @@ Feature: Bootstrap a Salt minion via the GUI with an activation key
   Scenario: Check events history for failures on SLES minion with activation key
     Given I am on the Systems overview page of this "sle_minion"
     Then I check for failed events on history event page
+
+  Scenario: Cleanup: Logout from API
```

Comment:
Before someone mention it, yes all the Cleanup scenarios to logout have the same name, because when we do the exact same actions in a scenario, I consider we don't need to bother to use a different name, and it's even better when we need to refactor that scenario and apply it to all the equivalent ones.